### PR TITLE
Add Microsoft.AspNetCore.App.Ref,3.1.8 & 3.1.10

### PR DESCRIPTION
**Pull request by dseefeld**
For https://github.com/dotnet/source-build/issues/1940

### CODE REVIEW

**Review by crummel — APPROVED** [review]

File: src/targetPacks/ILsrc/microsoft.aspnetcore.app.ref/3.1.10/[Content_Types].xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (dagood) [inline src/targetPacks/ILsrc/microsoft.aspnetcore.app.ref/3.1.10/[Content_Types].xml:1]: Trim this file (multiple projects) `[Content_Types].xml` and:
`_rels/.rels`
`package/services/metadata/core-properties/6d80a3452fce4876a8220ed215d32203.psmdcp`?

File: src/targetPacks/ILsrc/microsoft.aspnetcore.app.ref/3.1.8/Microsoft.AspNetCore.App.Ref.csproj [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputPath>$(ArtifactsBinDir)microsoft.netcore.app/3.1.8/</OutputPath>
+    <BaseIntermediateOutputPath>$(ArtifactsObjDir)microsoft.netcore.app/3.1.8/</BaseIntermediateOutputPath>
```
Comment (dagood) [inline src/targetPacks/ILsrc/microsoft.aspnetcore.app.ref/3.1.8/Microsoft.AspNetCore.App.Ref.csproj:5]: Wrong project name. Adopt new project style from text only packages that doesn't require manual fixups?

https://github.com/dotnet/source-build-reference-packages/blob/a72f1ee968104a9c13622e0da4ca42fa1fe9e523/src/textOnlyPackages/src/microsoft.dotnet.common.itemtemplates/3.1.8/Microsoft.DotNet.Common.ItemTemplates.csproj#L1-L12

**Review by dagood — CHANGES_REQUESTED** [review]

File: src/targetPacks/ILsrc/microsoft.aspnetcore.app.ref/3.1.10/[Content_Types].xml (hunk 1 above)
Comment (dseefeld) [reply src/targetPacks/ILsrc/microsoft.aspnetcore.app.ref/3.1.10/[Content_Types].xml:1]: Removed from all Microsoft.AspNetCore.App.Ref projects.

File: src/targetPacks/ILsrc/microsoft.aspnetcore.app.ref/3.1.8/Microsoft.AspNetCore.App.Ref.csproj (hunk 2 above)
Comment (dseefeld) [reply src/targetPacks/ILsrc/microsoft.aspnetcore.app.ref/3.1.8/Microsoft.AspNetCore.App.Ref.csproj:5]: Thanks.  Again, updated all projects under Microsoft.AspNetCore.App.Ref.

**Review by dagood — APPROVED** [review]

**Pull request merged**